### PR TITLE
Add Hasura Checks, Event Trigger, REST Endpoints

### PR DESCRIPTION
**Pull request by cif**
Still have to add REST endpoints but getting pipeline changes in now to test in GH

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -70,7 +68,28 @@ jobs:
...
-      - name: Smoke tests
+      - name: Run migrations on local container
         # TODO: check if hasura migrations + hasura ic list
         run: |
           DATABASE_URL=postgres://postgres:password@localhost:5432/minterop diesel migration run
+
+      - name: Apply hasura metadata to sandbox
+        run: |
+          npm i -g hasura-cli --y
+          hasura metadata apply --project hasura --admin-secret "${{ secrets.HASURA_SANDBOX_ADMIN }}" --endpoint https://interop-sandbox.hasura.app/
```
Comment (tifrel) [inline .github/workflows/ci.yaml:79]: Why not use SANDBOX_ENV?

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
-# minterop-data
+# Minterop Data
+
+## Postgres and Hasura GraphQL/REST API configuration for Mintbase Indexers
+
+Configurations here manage both **postgres database column migrations** and **hasura configuration** in a single script that can be run in one of three environments **sandbox (mainnet NEAR data)**, **testnet (testnet NEAR data)**, and **mainnet**.
+
+Running `scripts/redeploy.sh [env]` can be done locally but will require a {environment_name}.env file will need to be present. Ask if you need help with the contents of this file.
+
+**Note, local migrations should only ever be done run sandbox because they lack the inconsitency checks in CICD**
```
Comment (tifrel) [inline README.md:9]: Maybe we should include the `hasura metadata ic list` in the script, so we at least get a warning when we have problems on mainnet/testnet.

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+
+## Migrations
+
+Migrations under the hood use [diesel](https://diesel.rs/), a Rust library for database manipulation. You will need to install this via cargo to test migrations locally.
+
+<br>
```
Comment (tifrel) [inline README.md:15]: huh? :D

**Review by tifrel — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (cif) [reply .github/workflows/ci.yaml:79]: Saw that in secrets as well, this was just copied from branch on mintlake, if you feel strongly we should keep the whole env consistent will change it

File: README.md (hunk 3 above)
Comment (cif) [reply README.md:15]: Still no good way in markdown to add line breaks without using HTML

File: README.md (hunk 2 above)
Comment (tifrel) [reply README.md:9]: I can do that on next iteration if you leave me a TODO comment in there :)

**Pull request merged**